src/NzbDrone.Core/DataAugmentation/Scene/SceneMappingService.cs

```diff
@@ -66,6 +68,24 @@ public List<string> GetSceneNames(int tvdbId, List<int> seasonNumbers, List<int>
             return FilterNonEnglish(names);
         }
 
+        public List<SceneMapping> GetSceneMappings(int tvdbId, List<int> seasonNumbers)
+        {
+            var mappings = FindByTvdbId(tvdbId);
+
+            if (mappings == null)
+            {
+                return new List<SceneMapping>();
+            }
+
+            seasonNumbers = seasonNumbers.ToList();
```

Comment:
`seasonNumbers` is already a list.

Reply:
Yes, but I want to add one to it without modifying a list the caller might have.
But yeah, it's ugly code. I'll change it.